server/apiv2/handlers/tso_keyspace_group.go

```diff
@@ -40,9 +40,9 @@ func RegisterTSOKeyspaceGroup(r *gin.RouterGroup) {
 	router.GET("", GetKeyspaceGroups)
 	router.GET("/:id", GetKeyspaceGroupByID)
 	router.DELETE("/:id", DeleteKeyspaceGroupByID)
+	router.PATCH("/:id", SetNodesForKeyspaceGroup)          // only to support set nodes
+	router.PATCH("/:id/*node", SetPriorityForKeyspaceGroup) // only to support set priority
```

Comment:
why use *?

Reply:
Why do we change it? The current pd-ctl command "./pd-ctl keyspace-group set-priority 1 "http://pd-tso-server-0.tso-service.tidb-serverless.svc:2379" -- 100" seems good enough to me. With this change, how does the pd-ctl command look like?

Reply:
> Why do we change it? The current pd-ctl command "./pd-ctl keyspace-group set-priority 1 "http://pd-tso-server-0.tso-service.tidb-serverless.svc:2379" -- 100" seems good enough to me. With this change, how does the pd-ctl command look like?

This pr doesn't affect pd-ctl

Reply:
> Why do we change it? The current pd-ctl command "./pd-ctl keyspace-group set-priority 1 "http://pd-tso-server-0.tso-service.tidb-serverless.svc:2379" -- 100" seems good enough to me. With this change, how does the pd-ctl command look like?

It is a suggestion from @rleungx, which make url more restful.

POST means create and PATCH means partially update.

Reply:
fine-grained update is addressed by GraphQL, isn't it?